### PR TITLE
🗄 Add id fields to club's and event's domain & DTO definitions

### PR DESCRIPTION
**Pull request by KTong821**
## Purpose
This PR allows the backend to return unique ids for each object to the frontend that can then be used to query specific entities for more detail (yet to be implemented). 

Also removes the ClubId and EventId value objects. these were found to be essentially wrappers around UniqueEntityID objects with little value but necessitated code such as `club.id.id.toString()`. 

## Testing
Tests modified to account for new property; all tests pass.

### CODE REVIEW

**Review by xujustinj — DISMISSED** [review]

File: server/src/modules/clubs/mappers/club-map.ts [hunk 1]
```diff
@@ -4,37 +4,27 @@ import { EventEntity } from '../../../shared/infra/db/entities/event.entity'
 import { BasicEvent, Club } from '../domain/entities/club'
 import { ClubDTO } from './club-dto'
 
-const basicEventMap = (eventEntities: Array<EventEntity>): Array<BasicEvent> => {
-  const events = eventEntities.map((eventEntity) => {
+const basicEventToDomain = (eventEntities: Array<EventEntity>): Array<BasicEvent> =>
```
[the remark this answers is not in the document]
Comment (KTong821) [reply server/src/modules/clubs/mappers/club-map.ts:7]: basicEventstoDomain (added plural)?

[the remark this answers is not in the document]
Comment (xujustinj) [reply server/src/modules/clubs/mappers/club-map.ts:7]: yeah, that works!

```suggestion
const basicEventsToDomain = (eventEntities: Array<EventEntity>): Array<BasicEvent> =>
```

**Review by jeffzh4ng — COMMENTED** [review]
Looks good to me. I'll let @orizabal  give the final +1 though :)

**Review by orizabal — DISMISSED** [review]
looks good to me!

**Review by xujustinj — APPROVED** [review]

**Review by orizabal — APPROVED** [review]

**Pull request merged**